src/journal-remote/journal-remote-main.c

```diff
@@ -152,6 +155,24 @@ static int dispatch_http_event(sd_event_source *event,
                                uint32_t revents,
                                void *userdata);
 
+static int get_accept_encoding(char **encoding) {
```

Comment:
`get` sounds we obtain from encoding from the string.
Please rename this to e.g. `build_accept_encoding()` or so.

Also, rename `encoding` -> `ret`.